src/components/EmojiPicker/EmojiPicker.js

```diff
@@ -176,6 +186,8 @@ class EmojiPicker extends React.Component {
                 }}
                 anchorOrigin={this.state.emojiPopoverAnchorOrigin}
                 measureContent={this.measureContent}
+                outerStyle={Browser.isMobile() && {maxHeight: this.props.windowHeight, marginTop: this.props.viewportOffsetTop}}
```

Comment:
Why only apply this style to mobile browsers?

Reply:
On the web, we handle keyboards differently, in this case, this style is used to adjust content dimensions when the keyboard is opened.